src/components/work/utils.js

```diff
@@ -124,27 +124,28 @@ export function getSeo(work) {
 }
 
 export function useBranchUserAndHasDigitalAccess(selectedPids) {
-  const { loanerInfo } = useUser();
+  const user = useUser();
 
   const {
     data: branchUserData,
     isLoading: branchIsLoading,
     isSlow: branchIsSlow,
   } = useData(
     selectedPids &&
-      loanerInfo?.pickupBranch &&
+      user?.loanerInfo?.pickupBranch &&
       branchesFragments.branchDigitalCopyAccess({
-        branchId: loanerInfo?.pickupBranch,
+        branchId: user?.loanerInfo?.pickupBranch,
       })
   );
 
   const hasDigitalAccess = useMemo(() => {
     return (
       branchUserData?.branches?.result
         ?.map((res) => res.digitalCopyAccess === true)
-        .findIndex((res) => res === true) > -1
+        .findIndex((res) => res === true) > -1 &&
+      user?.authUser?.rights?.digitalArticleService
     );
-  }, [branchUserData, loanerInfo]);
+  }, [branchUserData, user]);
```

Comment:
Jeg kan godt være lidt bekymret for, at vi retter i "hasDigitalAccess", fordi den fortæller faktisk mere om om brugeren har digitalAccess, og ikke om hvad visningen skal være. For nu, så tror jeg hellere vi bør rette det mere lokalt ved f.eks. at sige `getAllowedAccessByType(hasDigitalAccess || !user?.authUser?.rights?.digitalArticleService)`.

- Længere ud i fremtiden: Men endnu mere konceptuelt, så kigger vi jo også på brugernes rettigheder nu og ikke branches mere, så man kunne gå endnu mere deep og give user med til `accessFactory()` og så løse alle problemerne derfra i stedet.